wbstatus.py

```diff
@@ -106,18 +107,20 @@ def get_user(self, phid):
         return retval
 
 
-# the TaskStore is a wrapper around the Phabricator manifest.query
-# API call, so this object indexes the result by task number.  This
-# is necessary because the Phabricator API inexplicably doesn't
-# return the result in such a way that the tasks can be easily
-# looked up by task number.
 class TaskStore(object):
+    """The TaskStore is a wrapper around the Phabricator manifest.query API
+    call, so this object indexes the result by task number.  This is necessary
+    because the Phabricator API inexplicably doesn't return the result in such
+    a way that the tasks can be easily looked up by task number.
+    """
 
     def __init__(self, tasknums=set()):
         self.tasknums = tasknums
 
     def load_from_phabricator(self, phab, cachedir):
-        taskquerycall = lambda: phab.maniphest.query(ids=self.tasknums)
```

Comment:
The changes from using lambdas to named functions breaks the script.  I haven't had a chance to debug it fully, but I suspect it's because the scope of "self" is different when the functions are assigned via anonymous function, and that declaring them this way screws up the closure.  The error I'm getting is:

Traceback (most recent call last):
  File "./wbstatus.py", line 512, in <module>
    main()
  File "./wbstatus.py", line 432, in main
    for tasknum, taskfeed in activity.iteritems():
AttributeError: 'NoneType' object has no attribute 'iteritems'